Dockerfile

```diff
@@ -359,6 +359,21 @@ RUN \
 # https://gitlab.alpinelinux.org/alpine/aports/-/issues/11806
 RUN sed -i 's/-lgcc_s//' /usr/local/lib/pkgconfig/rav1e.pc
 
+# bump: librtmp /LIBRTMP_COMMIT=([[:xdigit:]]+)/ gitrefs:https://git.ffmpeg.org/rtmpdump.git|re:#^refs/heads/master$#|@commit
+# bump: librtmp after ./hashupdate Dockerfile LIBRTMP $LATEST
+# bump: librtmp link "Commit diff $CURRENT..$LATEST" https://git.ffmpeg.org/gitweb/rtmpdump.git/commitdiff/$LATEST?ds=sidebyside
+ARG LIBRTMP_URL="https://git.ffmpeg.org/rtmpdump.git"
+ARG LIBRTMP_COMMIT=f1b83c10d8beb43fcc70a6e88cf4325499f25857
+RUN \
+  git clone "$LIBRTMP_URL" && \
+  cd rtmpdump && git checkout $LIBRTMP_COMMIT && \
+  # Patch/port librtmp to openssl 1.1
```

Comment:
I noticed that the ffmpeg package for alpine builds with gnutls, maybe that could be an option? but also noticed that gnutls for alpine does not have any gnutls* package with .a files hmm